delivery_ups_oca/models/ups_request.py

```diff
@@ -18,15 +18,16 @@ def __init__(self, carrier):
         self.username = self.carrier.ups_ws_username
         self.password = self.carrier.ups_ws_password
         self.default_packaging_id = self.carrier.ups_default_packaging_id
+        self.use_packages_from_picking = self.carrier.ups_use_packages_from_picking
         self.shipper_number = self.carrier.ups_shipper_number
         self.service_code = self.carrier.ups_service_code
         self.file_format = self.carrier.ups_file_format
         self.package_dimension_code = self.carrier.ups_package_dimension_code
         self.package_weight_code = self.carrier.ups_package_weight_code
         self.transaction_src = "Odoo (%s)" % self.carrier.name
-        self.url = "https://onlinetools.ups.com"
+        self.url = "https://wwwcie.ups.com"
```

Comment:
could you define these urls as ir.config_parameter ?
May be you may change them to sandbox

Reply:
In other integration modules from other carriers, it has always been defined in the code because the URLs do not change (nor do we want them to change), this change was necessary because they were previously incorrect.